cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/SchedulerServiceTest.java

```diff
@@ -143,7 +144,7 @@ public void testSchedulesAcrossNamespace() throws Exception {
                                                        programType, TIME_SCHEDULE_1));
 
     Id.Program programInOtherNamespace =
-      Id.Program.from(new Id.Application(new Id.Namespace("otherNamespace"), appId.getId()),
+      Id.Program.from(new Id.Application(new Id.Namespace("otherNamespace"), appId.getApplication()),
```

Comment:
To convert it in a similar format as `appId.workflow(AppWithWorkflow.SampleWorkflow.NAME).toId()`?